app/code/Magento/Inventory/Model/ResourceModel/SourceStockLink/Collection.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model\ResourceModel\SourceStockLink;
+
+use Magento\Inventory\Model\ResourceModel\SourceStockLink as ResourceSourceStockLink;
+use Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection;
+use Magento\Inventory\Model\SourceStockLink as SourceStockLinkModel;
+
+/**
+ * Resource Collection of SourceStockLink entities
+ * It is not an API because SourceStockLink must be loaded via Source entity only
```

Comment:
>  It is not an API because SourceStockLink must be loaded via Source entity only
  
it's not correct , because for StockSourceLink we will have dedicated services. 
We would not process link just through the Source entity.

So, we can call this Collection as API